cam_diagnostics/kessler_diagnostics.F90

```diff
@@ -0,0 +1,55 @@
+module kessler_diagnostics
+
+   use ccpp_kinds, only:  kind_phys
```

Comment:
I wonder if it would be worth adding a comment here noting that this is only for the Kessler physics suite.

Reply:
added comment.